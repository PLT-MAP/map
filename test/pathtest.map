func main(){
        Graph g=new Graph();
        Node nj=new Node();
        Node ny=new Node();
        Node losangeles= new Node({'temp':85,'humidity':'low'});
        Node boston= new Node({'temp':87,'humidity':'high'});
        Node paris= new Node({'temp':80});
        Node kansascity = new Node({'temp':40,'humidity':'low'});
        Node sanfransisco= new Node({'temp':30,'humidity':'low'});
        Node durham= new Node({'temp':21,'humidity':'low'});
        Node minneapolis= new Node({'temp':41});
        Node stpaul= new Node({'temp':50,'humidity':'low'});
        Node philly= new Node({'temp':82,'humidity':'low'});
        Node pitts= new Node({'temp':90});
        Node stpeters= new Node({'temp':100});
        g.add(losangeles);
        g.add(paris);
        g.add(durham);
        g.add(philly);
        g.add(pitts);
        g.add(stpeters);
        DirEdge wee= new DirEdge(nj,ny);
  		UnDirEdge lol = new UnDirEdge(boston,paris);
        UnDirEdge nynj = new UnDirEdge(ny,nj,{'distance':100});
    	DirEdge pittsphilly = new DirEdge(pitts,philly,{'distance':10});
		DirEdge pittsparis = new DirEdge(pitts,paris,{'distance':15});
    	DirEdge stpaulpitts = new DirEdge(stpaul,pitts,{'distance':40});
        g.addEdge(pittsphilly);
        g.addEdge(stpaulpitts);
        g.addEdge(pittsparis);
        print(losangeles);
        g.adjacent(losangeles,paris);
        g.path(losangeles, paris);
       	g.draw('lol.jpg');
  
}
