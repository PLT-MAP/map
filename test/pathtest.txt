def main():
    p = nx.MultiDiGraph()
	p.add_node(nj[0], nj[1])
