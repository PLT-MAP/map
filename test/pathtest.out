import networkx as nx
import sys
import matplotlib.pyplot as plt
def main():
    p = nx.MultiDiGraph()
    p.add_nodes_from([(nj[0], nj[1])])
if __name__ == '__main__': 
	try:
		main()
	except:
		print'Error:',sys.exc_info()[0]
		print'Resolve error before running again!'